Specs/DateTime/Spanish/DateTimePeriodExtractor.json

```diff
@@ -114,17 +114,6 @@
       }
     ]
   },
-  {
-    "Input": "Estaré afuera de 4pm a 5pm del 2017-6-6",
```

Comment:
This test seems a correct equivalent to `I'll be out from 4pm to 5pm of 2017-6-6`.